tuning/sft_trainer.py

```diff
@@ -286,23 +296,67 @@ def main(**kwargs):
         choices=["pt", "lora", None, "none"],
         default="pt",
     )
+    parser.add_argument(
+        "--exp_metadata",
+        type=str,
+        default=None,
```

Comment:
this probably needs help text. looks like it expects a json string?

Reply:
done